introduction.md

```diff
@@ -1,3 +1,9 @@
 Hello, my name is Robert. I'm a software engineer in test.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?